layers/meta-resin-beaglebone/recipes-bsp/u-boot/u-boot-ti-staging/0001-ti_armv7_common.h-Don-t-hardcode-dev-0-but-use-mmcde.patch

```diff
@@ -1,29 +0,0 @@
-From 9eecb4e109f62e2e3f4094f4597afb7b819bf231 Mon Sep 17 00:00:00 2001
-From: Andrei Gherzan <andrei@resin.io>
-Date: Fri, 4 Mar 2016 15:33:46 +0100
-Subject: [PATCH 1/2] ti_armv7_common.h: Don't hardcode dev 0 but use mmcdev
-
-Signed-off-by: Andrei Gherzan <andrei@resin.io>
-
-Upstream-Status: Pending
-
----
- include/configs/ti_armv7_common.h | 2 +-
- 1 file changed, 1 insertion(+), 1 deletion(-)
-
-diff --git a/include/configs/ti_armv7_common.h b/include/configs/ti_armv7_common.h
-index 6011cf8..2ca7762 100644
---- a/include/configs/ti_armv7_common.h
-+++ b/include/configs/ti_armv7_common.h
-@@ -61,7 +61,7 @@
- #define DEFAULT_MMC_TI_ARGS \
- 	"mmcdev=0\0" \
- 	"mmcrootfstype=ext4 rootwait\0" \
--	"finduuid=part uuid mmc 0:2 uuid\0" \
-+	"finduuid=part uuid mmc ${mmcdev}:2 uuid\0" \
```

Comment:
resinHUP is not known to work with 2.X anyway so the assumption is that it is not working at the moment as it was never tested.
Conclusion is that when resinHUP will be tested on beaglebone 2.X this PR will get the approval to be merged? It's not clear what is the next step to be done as part of this PR so that we merge this.